articles/azure-monitor/app/monitor-functions.md

```diff
@@ -76,6 +76,63 @@ Your Java functions might have slow startup times if you adopted this feature be
 
 [!INCLUDE [azure-monitor-app-insights-test-connectivity](../../../includes/azure-monitor-app-insights-test-connectivity.md)]
 
+#### Duplicate Logs
+
+If you are using log4j or logback to log to the console, then distributed tracing for Java Functions will store a duplicate copy of your log4j or logback logs. To avoid this, follow the workarounds shown below.
```

Comment:
```suggestion
If you are using log4j or logback to log to the console, then distributed tracing for Java Functions will create a duplicate copy of your log4j or logback logs sent to Application Insights. To avoid this, follow the workarounds shown below.
```